packages/kamino-sdk/src/Kamino.ts

```diff
@@ -185,6 +190,33 @@ export class Kamino {
     return await batchFetch(strategies, (chunk) => WhirlpoolStrategy.fetchMultiple(this._connection, chunk));
   }
 
+  async getAllStrategies(strategyFilters: StrategiesFilters): Promise<Array<WhirlpoolStrategy | null>> {
```

Comment:
I think the name of this method should be something like `getStrategiesWithFilter` as we already have getStrategies that returns all strategies and it might be confusing which one to use.